packages/core/lib/services/api.test.js

```diff
@@ -24,7 +26,7 @@ describe('ApiService', () => {
     const mnemonicId = `${Date.now()}`;
     const accountName = 'Test Account';
 
-    testAccount = keyring.addFromUri(`${phrase}`);
+    testAccount = keyring.addFromMnemonic(phrase, null, 'sr25519');
```

Comment:
do we need tests with other types than sr255519?

Reply:
Yes, would be good

Reply:
https://www.notion.so/dockteam/Wallet-sdk-code-review-25d2f2d7644c470ea5a1012a8cd7c871